### PR TITLE
Deploy VM as the sample app instead of a Sleep pod

### PR DESCRIPTION
**Pull request by kirankt**
Increase size of PVC to 5G to accommodate the VM example
Update README with instructions for VM centric deployment.

### CODE REVIEW

File: tools/demo.sh [hunk 1]
```diff
@@ -26,8 +27,12 @@ $_kubectl -n ember-csi wait --timeout=300s --for=condition=Ready pod/external-ce
...
-$_kubectl -n sample-project wait --timeout=300s --for=condition=Ready pod/busybox-sleep
-$_kubectl -n sample-project describe pods busybox-sleep | tail
-$_kubectl -n sample-project exec -it busybox-sleep  -- df -h | grep -B 1 /data
+# Deploy Kubevirt and install virtctl
+$_kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/$KUBEVIRT_VERSION/kubevirt-operator.yaml
+$_kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/$KUBEVIRT_VERSION/kubevirt-cr.yaml
+
+# Install virtctl
+mkdir -p _out/cmd/virtctl
+curl -L -o _out/cmd/virtctl/virtctl https://github.com/kubevirt/kubevirt/releases/download/$VERSION/virtctl-$KUBEVIRT_VERSION-linux-amd64
```
Comment (cschwede) [inline tools/demo.sh]: $VERSION needs to be $KUBEVIRT_VERSION

File: README.md [hunk 2]
```diff
@@ -12,9 +12,35 @@ This deployment deploys an ephemeral Ceph container alongside Ember CSI and its
...
-$ make deploy
-$ make demo
+$ make all
+```
+
+At this point we have  fully working cluster with KubeVirt and Ember-CSI deployed. We can now deploy VMs.
+
+```
+# Creating a virtual machine
+$ kubectl apply -f https://raw.githubusercontent.com/kubevirt/demo/master/manifests/vm.yaml
```
Comment (cschwede) [inline README.md]: This command only works for me if I replace kubectl with ./cluster/kubectl.sh  (same for all following commands).

File: README.md [hunk 3]
```diff
@@ -12,9 +12,35 @@ This deployment deploys an ephemeral Ceph container alongside Ember CSI and its
...
+
+# After deployment you can manage VMs using the usual verbs:
+$ kubectl get vms
+$ kubectl get vms -o yaml testvm
+
+# To start a VM you can use
+$ ./virtctl start testvm
```
Comment (cschwede) [inline README.md]: Should be ./cluster/virtctl.sh - "make all" was executed in the top level dir, so we need the leading prefix dir here and for all other commands

File: README.md [hunk 4]
```diff
@@ -12,9 +12,35 @@ This deployment deploys an ephemeral Ceph container alongside Ember CSI and its
 ```
 $ git clone https://github.com/embercsi/ember-csi-kubevirt.git
 $ cd ember-csi-kubevirt/
-$ make cluster-up
-$ make deploy
-$ make demo
+$ make all
+```
```
Comment (cschwede) [inline README.md:16]: I had to execute "source tools/env.sh" here as well to make it work

**Review by cschwede — CHANGES_REQUESTED** [review]
A few small changes were required to make this work. Please have a look at my comments.

The PVC is also not used in this example, I think we should change this too.

**Comment by kirankt** [comment]
I have a new commit which uses Ember-CSI provisioned PVC to create a cirros VM. Please give it a whirl and review. Thanks.

File: tools/demo.sh (hunk 1 above)
Comment (kirankt) [reply tools/demo.sh]: The new commit fixes this.

File: README.md (hunk 2 above)
Comment (kirankt) [reply README.md]: Since we are creating an alias, we can use that instead. The new commit uses this.

File: README.md (hunk 3 above)
Comment (kirankt) [reply README.md]: again, we will use the alias 'v' here.

**Comment by kirankt** [comment]
The examples/cirros* files were missing. Added them.

**Comment by cschwede** [comment]
LGTM - tested and works as expected.

**Pull request merged**